### PR TITLE
HPCC-17100 Fix esp.xml missing format information

### PR DESCRIPTION
**Pull request by rpastrana**
- Creates xls var for line break, and indent
- Output break and indent before every esp service and binding
- Cleans out extra whitespace in xslt source

Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [x] I have added tests to cover my changes.
  - [x] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [x] Scalability
  - [ ] Performance
  - [x] Security
  - [x] Thread-safety
  - [x] Premature optimization
  - [x] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by rpastrana** [comment]
@wangkx Please take a look.

File: initfiles/componentfiles/configxml/esp.xsl [hunk 1]
```diff
@@ -181,7 +183,15 @@
                     </xsl:call-template>
                 </xsl:variable>
                 <xsl:variable name="bindName" select="concat('WSESPControl_Binding_', $process)"/>
+                <xsl:value-of disable-output-escaping="yes" select="$break" />
+                <xsl:value-of disable-output-escaping="yes" select="$indent" />
+                <xsl:value-of disable-output-escaping="yes" select="$indent" />
+                <xsl:value-of disable-output-escaping="yes" select="$indent" />
```
Comment (wangkx) [inline initfiles/componentfiles/configxml/esp.xsl]: In my option, adding 'break' is good enough. Too many indent lines ...

**Comment by richardkchapman** [comment]
Typo "everty" in commit message

File: initfiles/componentfiles/configxml/esp.xsl (hunk 1 above)
Comment (wangkx) [inline initfiles/componentfiles/configxml/esp.xsl]: Although esp.xml is in bad format, ECLWatch shows it in a good XML format.

Comment (rpastrana) [inline initfiles/componentfiles/configxml/esp.xsl]: the indent is needed if we're going to format the file content explicitly. The fact that eclwatch formats the xml in the viewer makes me wonder if we should bother w/ this change at all...
@RussWhitehead your thoughts please...

Comment (RussWhitehead) [inline initfiles/componentfiles/configxml/esp.xsl]: IMHO if we can easily format the XML properly then we should.

Comment (richardkchapman) [inline initfiles/componentfiles/configxml/esp.xsl]: I agree

**Pull request merged**